[ELK] Explain how to customize logstash configuration (#1416)
Scalingo/documentation: _posts/platform/getting-started/2000-01-01-getting-started-with-elk.md

The result of the above search contains a document having with a field `name`
set to `Alenala` and a field `message` set to `Hi!`.

> ### Custom configuration for Logstash

Reviewer: nitpick:
```suggestion
### Custom Configuration for Logstash
```

> In the logstash configuration repository, clone from
> [Scalingo/logstash-boilerplate](https://github.com/Scalingo/logstash-boilerplate),
> a config directory is present. All the files in this folder will be copied in the
> logstash configuration directory at runtime, allowing you to customize exactly
> how your want logstash to run.

Reviewer: I find this paragraph hard to understand. Moreover, we already talk about cloning our boilerplate on line 41. Here is a rephrase suggestion, WDYT?

```suggestion
The cloned boilerplate used to deploy your application contains a `config` directory. 
All the files in this folder will be copied in the Logstash configuration directory at runtime, 
allowing you to customize exactly how your want Logstash to run.
```

> For instance, if you want to edit the logging behavior of logstash, edit `config/log4j2.yml`

Reviewer: nitpick:
```suggestion
For instance, if you want to edit the logging behavior of Logstash, edit `config/log4j2.yml`
```

```
git add config/log4j2.yml

> git commit -m "Update how logstash is logging"

Reviewer: nitpick:
```suggestion
git commit -m "Update how Logstash is logging"
```
